packages/ra-ui-materialui/src/layout/Loading.tsx

```diff
@@ -53,7 +59,6 @@ const Loading = props => {
 };
 
 Loading.propTypes = {
-    classes: PropTypes.object,
```

Comment:
You shouldn't remove this one. When `classes` is present in the props, useStyles() uses it to override the default styles.